Makefile

```diff
@@ -11,14 +11,24 @@ clean: ## Delete pyc files
 build: ## Build Docker image
 	@docker build -t "${DOCKER_TAG}" .
 
-.PHONY: tests
-tests: clean build ## Run tests in container
+.PHONY: docker-tests
+docker-ui-tests: clean build ## Run tests in container
 	@docker run \
 		--net="host" \
 		--env REDASH_SERVER_URL="${REDASH_SERVER_URL}" \
 		--mount type=bind,source="${CURDIR}",target=/home/user/src \
 		"${DOCKER_TAG}"
 
+.PHONY: ui-tests
+ui-tests: ## Run tests outside of container
+	@pipenv run pytest --driver=Firefox --server-url=http://localhost:5000 --verify-server-url --variables=variables.json --html=report.html
+
+.PHONY: setup-redash
+setup-redash: ## Setup redash instance
+	@docker-compose run --rm server create_db
+	@docker-compose run --rm postgres psql -h postgres -U postgres -c "create database tests"
+	@wget -q ${REDASH_SERVER_URL}setup --post-data="name=Ashley McTest&email=ashley@example.com&password=REPLACE ME&org_name=default" -O /dev/null
```

Comment:
Please add quotes here to avoid weird behavior when ``REDASH_SERVER_URL`` accidentally contains spaces.

``"${REDASH_SERVER_URL}setup"``